main/settings.py

```diff
@@ -185,7 +185,15 @@
     "django.contrib.auth.backends.ModelBackend",
 )
 
-SESSION_ENGINE = "django.contrib.sessions.backends.signed_cookies"
+SESSION_ENGINE = get_string(
+    "SESSION_ENGINE",
+    "django.contrib.sessions.backends.signed_cookies",
+    description=(
+        "The backend that supports user sessions "
+        "(https://docs.djangoproject.com/en/3.1/topics/http/sessions/#configuring-the-session-engine)"
+    ),
+    required=True,
```

Comment:
Can you make this `False` instead? It'll cause the app to fail to boot otherwise and there's a reasonable default.

Reply:
Misunderstanding on my end. Fixed